### ISSUE TITLE
[Feature Request] username and password

### ISSUE THREAD

**Issue by TonyStark** [adamdehaven/fetchurls#8]
suppose if site asking username password like http://usename:password@example.com
then how to fetch urls from that site?

**Comment by adamdehaven** [comment]
@TonyStark How about that for quick turnaround? 🎉 I was already working on some changes, so this was timely.

**Comment by TonyStark** [comment]
Thanks for quick improvement, i tried as you mentioned in readme, but seems not working
```
$bash fetchurls.sh -u admin -p pass123

Fetch a list of unique URLs for a domain.

Enter the full domain URL ( https://example.com )
Domain URL: https://admin.example.io/files

ERROR: 'https://admin.example.io/files' is unresponsive or is not a valid URL.
       Ensure the site is up by checking in your browser, then try again.


```

**Comment by adamdehaven** [comment]
@TonyStark Hmm... it worked in my tests; however, I didn't have an actual auth URL to test with. If you have something set up where I can try hitting the URL with credentials, I can take a look.

**Comment by TonyStark** [comment]
ok give me your email, i will try to manage for you
Edit: i can give you only site because i cant share login. so you can figure out why site showing unresponsive in ternimal

**Comment by adamdehaven** [comment]
@TonyStark adam@dehaven.org

**Comment by adamdehaven** [comment]
I passed `--username` to `wget` instead of `--user`. Fixing now... will update in a moment.

**Comment by adamdehaven** [comment]
Fixed implementation with [v3.1.1](https://github.com/adamdehaven/fetchurls/releases/tag/v3.1.1)

**Comment by TonyStark** [comment]
Thanks Adam,
Its working Fine :)
